packaging/os/apt.py

```diff
@@ -394,10 +401,15 @@ def install(m, pkgspec, cache, upgrade=False, default_release=None,
         else:
             autoremove = ''
 
+        if only_upgrade:
+            only_upgrade = '--only-upgrade'
+        else:
+            only_upgrade = ''
```

Comment:
I added the flag to the build_dep case, keeping the style it was done before (string interpolation).